testConfig.js

```diff
@@ -33,7 +33,10 @@ module.exports = ({files, path, testFile, singleRun}) => ({
             istanbul: { noCompact: true }
         }
     },
-
+    browserConsoleLogOptions: {
+        terminal: true,
+        level: 'DISABLE'
```

Comment:
Where did you found this option? I see LOG_DISABLE [here](https://github.com/karma-runner/karma/blob/eaeffe94470775fdc8b9924e386a96fb8893ac47/docs/config/01-configuration-file.md#loglevel)

Reply:
[here](https://github.com/karma-runner/karma/blob/eaeffe94470775fdc8b9924e386a96fb8893ac47/docs/config/01-configuration-file.md#browserconsolelogoptions)

Reply:
I've seen the `browserConsoleLogOptions`. I meant `level: 'DISABLE'` instead of options.LOG_DISABLE. I'm sure that @mbarto used the right value, but I'd like to check from where this string comes.